src/core_modules/capture-core/components/ListView/Filters/FilterButton/ActiveFilterButton.component.js

```diff
@@ -22,6 +26,7 @@ const getStyles = (theme: Theme) => ({
     },
     clearIcon: {
         color: theme.palette.text.secondary,
+        marginTop: spacers.dp4,
```

Comment:
What was the reason for adding this margin? Now it looks to me that the clear icon is a bit lower than the chevron and the filter text 🤔 
- with marginTop

<img width="221" alt="Screenshot 2024-09-05 at 10 39 39" src="https://github.com/user-attachments/assets/37c24626-a11f-4913-b3f8-02e343f00efd">

- without marginTop

<img width="224" alt="Screenshot 2024-09-05 at 10 42 25" src="https://github.com/user-attachments/assets/3c39e4b9-8a2e-4fd5-9bda-8388526db88f">